### DISCUSSION TITLE
Spectrogram editing?

### DISCUSSION THREAD

**Discussion by jamesAllenJahner3rd** [audacity/audacity#1210] [Ideas & Feature Requests]
I'm Not a programmer.
I noticed cat bell ringing while I was talking in my Spectrogram and used some of the plugins to remove it and thought; what if there was a brush that I could visually remove the sound. or edit it visually.

**Comment by Paul-Licameli** [comment]
Funny you should mention that... watch this student's summer project that I'm advising!  https://github.com/audacity/audacity/pull/1193

**Reply by jamesAllenJahner3rd** [reply]
well snap, how do i download the win 64 plugin :) I can't wait to give it a try.
